doctests/dt_hash.py

```diff
@@ -0,0 +1,91 @@
+# EXAMPLE: hash_tutorial
+# HIDE_START
+import redis
+
+r = redis.Redis(decode_responses=True)
+# HIDE_END
+# STEP_START set_get_all
+res1 = r.hset(
+    "bike:1",
+    mapping={
+        "model": "Deimos",
+        "brand": "Ergonom",
+        "type": "Enduro bikes",
+        "price": 4972,
+    },
+)
+print(res1)  # 4
+
+res2 = r.hget("bike:1", "model")
+print(res2)  # 'Deimos'
+
+res3 = r.hget("bike:1", "price")
+print(res3)  # '4972'
+
+res4 = r.hgetall("bike:1")
+print(
```

Comment:
I recommend a slightly different format for showing expected replies https://github.com/redis/redis-py/blob/emb-examples/doctests/search_quickstart.py#L242
It allows us to avoid such weird formatting for short print statements.